test-five/react-app-cdk/.github/workflows/cdk-deploy.yml

```diff
@@ -0,0 +1,53 @@
+name: CDK Deploy
```

Comment:
I think the .github folder needs to be at the root of your project which might explain why you can't get it to trigger

Reply:
ah yeah that would make sense